lib/rafka.rb

```diff
@@ -22,28 +22,4 @@ def self.wrap_errors
     raise ConsumeError, e.message[5..-1] if e.message.start_with?("CONS ")
     raise CommandError, e.message
   end
-
-  # redis-rb until 3.2.1 didn't retry to connect on
-  # Redis::CannotConnectError (eg. when rafka is down) so we manually retry
-  # 5 times
-  #
-  # TODO(agis): get rid of this method when we go to 3.2.1 or later, because
-  # https://github.com/redis/redis-rb/pull/476/
-  def self.with_retry(times: REDIS_DEFAULTS[:reconnect_attempts], every_sec: 1)
-    attempts = 0
-
-    begin
-      yield
-    rescue Redis::CannotConnectError => e
-      if Gem::Version.new(Redis::VERSION) < Gem::Version.new("3.2.2")
-        if attempts < times
-          attempts += 1
-          sleep every_sec
```

Comment:
The new implementation does not sleep for exactly `1s`. It sleeps for way less and increases it at each attempt.

Maybe you want to adjust the defaults

Reply:
Maybe you are talking about redis-rb 4.x? We depend on 3.3.x here.

Reply:
Nvm my previous comment. I've set this properly in the defaults.